### PR TITLE
Gujarati translations - Updated till C programming Language #113

### PR DESCRIPTION
**Pull request by shahkv95**
## Title - Updated the Gujarati translations of all words before the keyword section of the C language

This pull request (PR) is made in reference to #113

## Description
- Updated and added new translations of all the words before the start of the Language C - keywords - i.e. line 326. 
- This PR includes the updates made that include comprehensive review Gujarati translations of all words before the keyword section of C language
- I have created a standard format where I request everyone contributor to add the translations with a hyphen with more than one word within a translation. 

## Motivation and Context
The project itself is an intriguing idea and the main motivation to create this PR was to review the changes from other contributors and update them. As for the fact that Gujarati isn't a language that resembles the semantics of English and it persists with the regional meanings. To avoid confusion, I have updated the translations with the most common meanings and most appropriate meanings associated with the computing language. This change will improve the existing translations and solve the issue mentioned above.

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented on my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
- [] New and existing unit tests pass locally with my changes
- [X] Any dependent changes have been merged and published in downstream modules

### PTR
I will review the remaining changes and keep updating the translations iteratively.

### CODE REVIEW

**Comment by shahkv95** [comment]
@madiedgar Could you please review the changes?
Thank you!

**Review by shahkv95 — COMMENTED** [review]
- Updated few translations
- Added hyphen between translations having space-separated words
- Added new translations

File: locale/gu.yml [hunk 1]
```diff
@@ -16,75 +16,98 @@ language-abbreviation: "gu"
...
   INFO: "માહિતી"
     # ✅ @179priyasoni
     # ✅ @aadii0408
+    # ✅ @shahkv95
   IDEA: "વિચાર"
     # ✅ @179priyasoni
     # ✅ @aadii0408
-  DEBUG: ""
+    # ✅ @shahkv95
+  DEBUG: "ખામી-સુધારવી"
```
Comment (madiedgar) [inline locale/gu.yml:28]: I need to update the documentation to reflect this - if the first translator does not enter anything into the initial space, we will keep it blank. This is so we can better track the progression of reviews. 👍  I will update this for you as the documentation did not describe how to do this correctly.

**Review by madiedgar — APPROVED** [review]
Thank you!

File: locale/gu.yml (hunk 1 above)
Comment (madiedgar) [reply locale/gu.yml:28]: @shahkv95 it seems I do not have permissions to push to your repo, so I cannot make my changes in this PR. Forgive my work around 👍

**Comment by shahkv95** [comment]
I have updated the permissions - now you will be able to edit as well in my upcoming PRs @madiedgar

**Pull request merged**